File: 2-BMI-calculator-extended/app/src/main/java/com/example/bmi_calc_extended/Chart.java
package com.example.bmi_calc_extended;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Base64;
import android.webkit.WebSettings;
import android.webkit.WebView;

public class Chart extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_chart2);

        WebView webView = (WebView) findViewById(R.id.webview);

        WebSettings webSettings = webView.getSettings();
        webSettings.setJavaScriptEnabled(true);

//        webView.loadUrl("https://www.journaldev.com/");
        String unencodedHtml =
                "<html><body><iframe width=\"100%\" height=\"480\" frameborder=\"0\" src=\"https://koronawirusunas.pl/u/polska\"></iframe> </body></html>";
        String encodedHtml = Base64.encodeToString(unencodedHtml.getBytes(),
                Base64.NO_PADDING);
        webView.loadData(encodedHtml, "text/html", "base64");
    }
}

File: BeHealthyKotlinApp/app/src/main/java/com/example/behealthykotlinapp/Chart.kt
package com.example.behealthykotlinapp

//import android.R
import android.os.Bundle
import android.util.Base64
import android.view.View
import android.webkit.WebView
import androidx.appcompat.app.AppCompatActivity


class Chart : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_chart)

        val webView = findViewById<View>(R.id.webview) as WebView

        val webSettings = webView.settings
        webSettings.javaScriptEnabled = true

        val unencodedHtml =
            "<html><body><iframe width=\"100%\" height=\"480\" frameborder=\"0\" src=\"https://koronawirusunas.pl/u/polska\"></iframe> </body></html>"
        val encodedHtml: String = Base64.encodeToString(
            unencodedHtml.toByteArray(),
            Base64.NO_PADDING
        )
        webView.loadData(encodedHtml, "text/html", "base64")
    }
}